lib/grape/validations/validators/values.rb

```diff
@@ -17,12 +19,11 @@ def validate_param!(attr_name, params)
         excepts = @excepts.is_a?(Proc) ? @excepts.call : @excepts
         param_array = params[attr_name].nil? ? [nil] : Array.wrap(params[attr_name])
 
-        if !param_array.empty? && param_array.all? { |param| excepts.include?(param) }
-          raise Grape::Exceptions::Validation, params: [@scope.full_name(attr_name)], message: except_message
-        end
+        raise Grape::Exceptions::Validation, params: [@scope.full_name(attr_name)], message: except_message \
```

Comment:
I could do that for the `values` check (the second one) but not for the `excepts` check (the first one) since I don't want a missing `excepts` to skip the `values` check. But, honestly, I kind of like the symmetry of the two checks -- I think it makes them a little quicker to read and understand.  Your call, though.  If you want me to switch the second one to a `return unless` I'll be happy to do that.  Please advise.